pkgs/development/libraries/apr-util/default.nix

```diff
@@ -23,7 +23,7 @@ stdenv.mkDerivation rec {
 
   configureFlags = ''
     --with-apr=${apr} --with-expat=${expat}
-    --with-crypto
+    ${if !stdenv.isCygwin then "--with-crypto" else "--without-pgsql --without-sqlite2 --without-sqlite3 --without-freetds --without-berkeley-db --without-crypto"}
```

Comment:
Oh, I wasn't commenting on the particular flags. Just that the expression level involved triple negation (if not not cygwin, then turn off pgsql) :smile: 

`if stdenv.isCygwin then "--without-pgsql ..." else "--with-crypto"` would be a little less convoluted, but that's a pretty minor point!